app/config.py

```diff
@@ -36,6 +36,7 @@ class Settings(BaseSettings):
     postgres_password: str
     postgres_host: str
     postgres_port: int = 5432
+    cors_allow_origins: list = []
```

Comment:
shouldn't you use a Field with a default factory here?
https://docs.pydantic.dev/latest/concepts/fields/#default-values
And you can use list[str]

Reply:
yes you're right, I fixed it here : https://github.com/openfoodfacts/open-prices/pull/62/commits/0fa085bd2cf76d3fe3af70420d03e0ba06766e06

Reply:
not yet a pro of these typings in python ^^